### ISSUE TITLE
Remove eslint override in RadioButtonGroup.js

### ISSUE THREAD

**Issue by bradfordjanice** [hackoregon/civic#784]
## Description

Because a dependency was missing it was necessary to use
`// eslint-disable-next-line import/no-extraneous-dependencies`
in the file RadioButtonGroup.js to prevent linting errors.

This line will no longer needed after #721 is merged.

### What should happen?

Remove this line of code from RadioButtonGroup.js after #721 is merged.
